assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/BaseAssignmentService.java

```diff
@@ -10764,7 +10761,7 @@ public BaseAssignmentSubmission(String id, String assignId, String submitterId,
 			m_assignment = assignId;
 			m_properties = new BaseResourcePropertiesEdit();
 			addLiveProperties(m_properties);
-			m_submitters = new ArrayList();
+			m_submitters = new ArrayList<String>();
```

Comment:
+1 for using language features. Since Java 8 is required for 11+, I'm all in for using functionality that simplifies our code base!